gui/inputs/confidence.py

```diff
@@ -27,4 +26,4 @@ def render_confidence_level_widgets(self, r):
             .grid(sticky='w', row=r, column=2)
 
     def modify_confidence_level(self, confidence_lvl):
-        self.bot.setConfidence(confidence_lvl * .01)
+        self.container.bot.setConfidence(confidence_lvl * .01)
```

Comment:
new update, access bot like this instead of having an own separate bot reference in the class.